pkg/cmd/server/api/v1/types.go

```diff
@@ -702,93 +702,100 @@ type LDAPSyncConfig struct {
 	api.TypeMeta `json:",inline"`
 	// Host is the scheme, host and port of the LDAP server to connect to:
 	// scheme://host:port
-	Host string `json:"host" description:"scheme://host:port for the LDAP server"`
+	URL string `json:"url" description:"scheme://host:port for the LDAP server"`
```

Comment:
We've inherited the godoc-to-swagger generator already, so yes, no descriptions needed.